app/scripts/controllers/landingPage.js

```diff
@@ -51,10 +52,20 @@ angular.module('openshiftConsole')
 
     AuthService.withUser().then(function() {
       var includeTemplates = !_.get(Constants, 'ENABLE_TECH_PREVIEW_FEATURE.template_service_broker');
-      Catalog.getCatalogItems(includeTemplates).then(function(items) {
+      Catalog.getCatalogItems(includeTemplates).then(_.spread(function(items, errorMessage) {
+        if (errorMessage) {
+          var alert = {
+            name: 'error-loading-catalog-items',
+            data: {
+              type: 'error',
+              message: errorMessage
+            }
+          };
+          AlertMessageService.addAlert(alert);
```

Comment:
Probably best to have duplicates rather than none at all. I will add the notification back here, remove the alert here and file an issue for catalog to not add notifications.